src/creatures/players/player.cpp

```diff
@@ -6722,6 +6722,10 @@ std::pair<std::vector<std::shared_ptr<Item>>, std::map<uint16_t, std::map<uint8_
 				continue;
 			}
 
+			if (item.get()->isStoreItem()) {
```

Comment:
You don't need to get the raw pointer

```suggestion
			if (item->isStoreItem()) {
```

Reply:
Thanks, i am removed the raw pointer in the last commit.